src/rgw/rgw_admin.cc

```diff
@@ -3057,6 +3058,7 @@ int main(int argc, const char **argv)
 
   rgw_user user_id;
   string tenant;
+  string ns;
```

Comment:
suggest user_ns to distinguish from other namespacing (multi and other)

Reply:
ok